src/sfizz/Region.cpp

```diff
@@ -1172,12 +1173,27 @@ bool sfz::Region::parseLFOOpcodeV2(const Opcode& opcode)
     case hash("lfo&_phase"):
         lfo.phase0 = opcode.read(Default::lfoPhase);
         break;
+    case_any_ccN("lfo&_phase"):
+        processGenericCc(opcode, Default::lfoPhaseMod, ModKey::createNXYZ(ModId::LFOPhase, id, lfoNumber));
+        break;
     case hash("lfo&_delay"):
         lfo.delay = opcode.read(Default::lfoDelay);
         break;
+    case hash("lfo&_delay_oncc&"):
+        if (opcode.parameters.back() > config::numCCs)
+            return false;
+
+        lfo.delayCC[opcode.parameters.back()] = opcode.read(Default::lfoDelayMod);
+        break;
     case hash("lfo&_fade"):
         lfo.fade = opcode.read(Default::lfoFade);
         break;
+    case hash("lfo&_fade_oncc&"):
+        if (opcode.parameters.back() > config::numCCs)
+            return false;
+
+        lfo.fadeCC[opcode.parameters.back()] = opcode.read(Default::lfoFadeMod);
```

Comment:
That one possibly is dynamic, is it checked it in ARIA?
also, the fade possibly to change to linear.

Reply:
Huh, no I didn't but since there are no `smooth/step` version in sfzformat I assumed it was a straightforward one (like offset and the like). I'll check.

Reply:
> also, the fade possibly to change to linear.

What do you mean?

Reply:
just checked: linear and non-dynamic

Reply:
> What do you mean?

Nvm misremembered, I thought this one was still using exponential.